.github/workflows/comment-on-closed-waiting-issues.yml

```diff
@@ -0,0 +1,18 @@
+name: Comment on closed Waiting for Response Issues
+on:
```

Comment:
**Needs to be fixed**

This is suppose to add a comment when an issue gets closed and has the `Waiting for Response` label. But it's not working when the bot closes the issue. (it only works when a person closes the issue)